src/elements/ItemsContainerElement.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FunctionComponent } from 'react';
+
+const createElement = ({ id, className }: IProps) => ({
+    __html: `<div
+        is="emby-itemscontainer"
+        ${id}
+        class="${className}"
+    >
+    </div>`
+});
+
+type IProps = {
+    id?: string;
+    className?: string;
+}
+
+const ItemsContainerElement: FunctionComponent<IProps> = ({ id, className }: IProps) => {
```

Comment:
You can use the shorthand FC, it's the same thing.

Reply:
If you define Iprops in the generic FunctionComponent (FC), arguments should be automatically infered by it, so i shouldn't be needed a second declaration right after the first argument.

Reply:
@vitorsemeano https://github.com/jellyfin/jellyfin-web/pull/3792/commits/2b40dbd50f39f56924b0d7ee6a3cab2dab38cbe4